docs/integrator-guide/workflow-using-ecs.md

```diff
@@ -0,0 +1,89 @@
+## Performance Tuning, Cost Management, and Ingest Monitoring using ECS
+
+## Performance Tuning and Throttling
+
+There are several bottlenecks to the data ingest process. The optimal rate of ingest is a difficult balancing act, as ingesting too slowly means the ingest takes a large amount of calendar time, but ingesting too quickly risks overwhelming external systems that are accessed.
+
+The two primary external limitations to throughput are copying assets from a DAAC to S3 and creating granules in CMR. In general, copying from a DAAC to S3 is slow with respect to typical data access speeds, and much more slow than intra-S3 transfers. This a challenge with imagery data files that are often very large, and further compounded when there is a large variability among data file sizes in a group (e.g., some are megabytes in size and take seconds to transfer, and others are gigabytes in size and take tens of minutes to transfer). Additionally, some DAAC servers have restrictions on how many simultaneous downloads are allowed per-IP.
+
+Another limitation is when ingested granules are created in a system-specific CMR, which may be deployed using a low-scalability queue between the web service interface and the data store, restricting the maximum ingest rate.
+
+Techniques to manage these limitations are discussed in the following sections.
+
+### Sync Granule Performance
+
+By default, the SyncGranule component is deployed as an AWS Lambda. Lambdas provide very flexible compute resources, such that one only pays for the amount of memory-hours they use. However, in the default configuration in Cumulus, they suffer from two major drawbacks:
+
+1. Cumulus Lambdas are deployed within the private subnet of a VPC. This requires that their network traffic be routed through a NAT Gateway in a public subnet. All data retrieved by the Lambda from a DAAC web server will be charged at a rate of $0.045/GB. A typical bulk ingest can be petabytes of data, costing thousands of dollars, and even small test runs can cost hundreds of dollars. Lambdas deployed into public (instead of private) subnets cannot access the internet due to AWS networking constraints. Deploying them outside a VPC is a solution, though this is not currently supported by Cumulus.
```

Comment:
We don't really put NGAP/DAAC specific docs in the public docs as a rule of thumb, but to explain: NGAP currently has a NAT instance, not a NAT gateway, so there is not the additional cost for the NAT Gateway. If the source data is in S3 in the same region it would go through the VPC Endpoint and not the NAT.